pyveg/src/download_modules.py

```diff
@@ -126,11 +125,11 @@ def prep_data(self, date_range):
         -------
         url_list:  a list of URLs from which zipfiles can be downloaded from GEE.
         """
-        region = get_region_string(self.coords, self.region_size)
+        region = get_region_string(self.bounds)
         start_date, end_date = date_range
 
         image_coll = ee.ImageCollection(self.collection_name)
-        geom = ee.Geometry.Point(self.coords)
+        geom = ee.Geometry.Point(get_coords(self.bounds))
```

Comment:
We can specify the coordinate reference system in this line (https://developers.google.com/earth-engine/apidocs/ee-geometry-point).

I _think_ if we do that and keep everything in BNG then we might get more consistent download image dimensions.

Reply:
It worked :)